packages/v4/patternfly-docs/components-data.json

```diff
@@ -61,7 +61,7 @@
   },
   "calendar-month": {
     "illustration": "./images/component-illustrations/calendar-month.png",
-    "summary": "A <b>calendar month</b> component allows users to select and navigate between days, months and/or years. This component is usually used with the date picker component, to display date options and selections. For more information about usage, view <Link to='/components/date-picker/design-guidelines'>date picker guidelines</Link>."
+    "summary": "A <b>calendar month</b> component allows users to select and navigate between days, months and/or years. This component is usually used with the date picker component, to display date options and selections. For more information about usage, view <Link to='/components/date-and-time/date-picker'>date picker guidelines</Link>."
```

Comment:
I think this url should be `/components/date-and-time/date-picker//design-guidelines`